src/components/Questionnaire/QuestionTypes/MedicationRequestQuestion.tsx

```diff
@@ -973,17 +972,20 @@ const MedicationRequestGridRow: React.FC<MedicationRequestGridRowProps> = ({
         <Label className="mb-1.5 block text-sm lg:hidden">
           {t("authored_on")}
         </Label>
-        <DateTimePicker
+        <Input
+          type="datetime-local"
           value={
             medication.authored_on
-              ? new Date(medication.authored_on)
-              : undefined
+              ? new Date(medication.authored_on).toISOString().slice(0, 16)
```

Comment:
let's not use slice, this will ignore timezone information and potentially yield incorrect results